README.md

```diff
@@ -49,22 +48,12 @@ Install Python dependencies:
 
 Install client side dependencies:
 
-    npm install -g browserify
-    npm install
-
-If you plan to do client side JS developent, you will want to install [Watchify](https://github.com/substack/watchify):
-```
-npm install -g watchify
-```
+    $ npm install
 
 ### Configuration
 
 The Flask app talks to an API for data. See [openFEC](http://github.com/18F/openFEC).
 
-The app has HTTP auth enabled. You will need to set environment variables with your desired username and password.
-Those vars are `FEC_WEB_USERNAME` and `FEC_WEB_PASSWORD`. There are other config environment variables that you
-can set, but that have defaults. You can see those in `openfecwebapp/config.py`.
-
```

Comment:
The basic auth is being removed as a part of this PR, thus negating the need for these environment variables?